plugins/services/services_inspector.rb

```diff
@@ -106,8 +118,41 @@ def inspect_upstart_services
     services += disabled.map(&:first).each.map do |name|
       Service.new(name: name, state: "disabled")
     end
+  end
 
-    services.sort_by(&:name)
+  def parse_ubuntu_sysv
+    # Get all sysV services
+    out,err = @system.run_command(
+      "/usr/sbin/service",
+      "--status-all",
+      stdout: :capture, stderr: :capture)
+    services_output = out + err
+
+    sysv_all = services_output.each_line.map { |line|
+      line.chomp.sub(/^.*\]../, '')}
```

Comment:
Expression at 132, 36 should be on its own line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.